pyproject.toml

```
@@ -1,3 +1,55 @@
 [build-system]
-requires = ["setuptools"]
-build-backend = "setuptools.build_meta"
\ No newline at end of file
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "stable-audio-tools"
+version = "0.0.20"
+authors = [
+    {name = "Stability AI"}
+]
+description = "Training and inference tools for generative audio models from Stability AI"
+readme = "README.md"
+requires-python = ">=3.8"
```

When using uv sync, I needed python 3.10 specified here.

```suggestion
requires-python = ">=3.10"
```